ex28.txt

```diff
@@ -0,0 +1,24 @@
+True and True: True
```

Comment:
Its just an answer for the given exercise 28. I've execute the program in the python shell.